components/tooltip/tooltip.js

```diff
@@ -269,7 +269,6 @@ class Tooltip extends RtlMixin(LitElement) {
 		this.disableFocusLock = false;
 		this.forceShow = false;
 		this.offset = pointerRotatedOverhang + pointerGap;
-		this.showing = false;
```

Comment:
For sake of clarity for anyone reading this later, is this because the `showing` setter gets called, which in turn calls `_showingChanged` which calls `setAttribute` for `aria-hidden`, which would not be allowed in the constructor for custom element?

Reply:
Yeah, the defect appeared in `v1.42.1` when the `setAttribute('aria-hidden', 'false')` call was added to `_showingChanged`. `showing` and `state` were both reflected and set in the constructor but don't cause any issues on their own.